### PR TITLE
Add rate limit field to remote api.

### PR DESCRIPTION
**Pull request by newswangerd**

### CODE REVIEW

File: galaxy_ng/app/migrations/0018_set_rate_limit_default.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    # rate limit. This breaks cloud.redhat.com during collection syncs. This sets the
+    # default to 8, which should be low enough to not break all of our customer's
+    # collection syncs.
+    CollectionRemote.objects.all().update(rate_limit=8)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('galaxy', '0017_populate_repos_and_remotes'),
```
[the remark this answers is not in the document]
Comment (rochacbruno) [reply galaxy_ng/app/migrations/0018_set_rate_limit_default.py:16]: https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/app/migrations/0017_populate_repos_and_remotes.py#L5

[the remark this answers is not in the document]
Comment (newswangerd) [reply galaxy_ng/app/migrations/0018_set_rate_limit_default.py:16]: > @newswangerd what about updating the 0017 migration to add the same default to the Remotes on the creation?

This has to set the default on all remotes, not just newly created ones, so I think it has to be a different migration

[the remark this answers is not in the document]
Comment (rochacbruno) [reply galaxy_ng/app/migrations/0018_set_rate_limit_default.py:16]: @newswangerd got that, was just thinking if, for consistency, it was not a good idea to add the same data to the creation migration.

[the remark this answers is not in the document]
Comment (rochacbruno) [reply galaxy_ng/app/migrations/0018_set_rate_limit_default.py:16]: BUt that can be done in a different PR :)

**Review by rochacbruno — APPROVED** [review]

File: galaxy_ng/app/migrations/0018_set_rate_limit_default.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+from django.db import migrations
+
+def set_rate_limit_default(apps, schema_editor):
+    CollectionRemote = apps.get_model('ansible', 'CollectionRemote')
+
+    # By default pulp core sets the new rate_limit field to null, which remotes the
```
Comment (awcrosby) [inline galaxy_ng/app/migrations/0018_set_rate_limit_default.py:6]: s/remotes/removes

Curious, if the user sets the rate limit to zero, does that also effectively remove it?

**Review by awcrosby — APPROVED** [review]

File: galaxy_ng/app/migrations/0018_set_rate_limit_default.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (newswangerd) [reply galaxy_ng/app/migrations/0018_set_rate_limit_default.py:16]: I can add the default to 0017 in this PR. It's a quick fix :)

File: galaxy_ng/app/migrations/0018_set_rate_limit_default.py (hunk 2 above)
Comment (newswangerd) [reply galaxy_ng/app/migrations/0018_set_rate_limit_default.py:6]: Setting it to 0 does remove the rate limit

**Review by rochacbruno — APPROVED** [review]

**Pull request merged**